hi-ml-cpath/testhisto/testhisto/utils/test_report_utils.py

```diff
@@ -212,7 +215,9 @@ def test_collect_hyperdrive_metrics(metrics_df: pd.DataFrame, tmp_path: Path, ov
 
 @pytest.mark.parametrize('maximise', [True, False])
 def test_get_best_epochs(metrics_df: pd.DataFrame, maximise: bool) -> None:
-    best_epochs = get_best_epochs(metrics_df, 'val/accuracy', maximise=maximise)
+    max_epochs_dict = {0: 3, 1: 3, 3: 3}
```

Comment:
We seem to be missing two test cases for the fixes introduced in this PR:
- Interrupted run with higher `max_epochs` than available values.
- Failed run with no values at all (`None`).

One idea to implement these is to adapt the existing `metrics_df` fixture. Thoughts?

Reply:
Sounds good. I have adapted the fixtures (`metrics_df` and `max_epochs_dict`) to consider the two test cases.